portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,6 +13,10 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.sps.data.Task;
```

Comment:
nit: please always sort the import alphabetically

Reply:
Oh okay, sound good, thank you!